app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesViewImpl.kt

```diff
@@ -5,17 +5,23 @@ import android.support.v7.widget.LinearLayoutManager
 import android.support.v7.widget.RecyclerView
 import android.view.View
 import butterknife.BindView
+import com.bluelinelabs.conductor.RouterTransaction
 import com.exallium.todoapp.R
 import com.exallium.todoapp.app.TodoApp
 import com.exallium.todoapp.mvp.BaseViewImpl
+import com.exallium.todoapp.notedetail.NoteDetailViewImpl
+import com.exallium.todoapp.screenbundle.ScreenBundleHelper
 import com.jakewharton.rxbinding.view.clicks
 import rx.Observable
 import timber.log.Timber
+import javax.inject.Inject
 
 /**
  * Conductor Controller displaying all Notes
  */
 class AllNotesViewImpl(bundle: Bundle) : BaseViewImpl<AllNotesView, AllNotesPresenter, AllNotesViewImpl, AllNotesComponent>(bundle), AllNotesView {
+    @Inject
+    lateinit var screenBundleHelper: ScreenBundleHelper
```

Comment:
Should never inject into a View.  Actually, besides what's in the base view and activity, we should never need to use the inject annotation